### PR TITLE
docs: add recipe for multi-write

### LINKED ISSUE #23: Add recipe for multi-write.
## Issue Overview

Add a recipe for the multi-write feature, you can put it under the Write (`sendAsync`) recipe.

## Proposed Solutions or Ideas

- Your documentation should be clear and concise.
- Use images where appropriate
- Use proper styling and hierarchy for text
- Use proper styling for code and variable names.
- Use proper callouts when required

Refer to docusaurus for more info: https://docusaurus.io/

### PR DESCRIPTION
**Pull request by Kaminar-i**
# Add recipe for multi-write

Fixes #23

## Types of change

- [ ] Bug
- [ ] Enhancement
- [x] Documentation

### CODE REVIEW

**Comment by Nadai2010** [comment]
Hi @Kaminar-i it's been a great job, on my part there are small details to correct, check them and we await the final review.

Thank you

File: docs/recipes/MultiWriteFeature.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+sidebar_position: 5
+title: Starknet Multi-Write Feature Recipe
```
Comment (metalboyrick) [inline docs/recipes/MultiWriteFeature.md:3]: Can change the title to "Batching multiple writes to a single transaction"

**Review by metalboyrick — CHANGES_REQUESTED** [review]
Minor changes and you're good, liked your example!

**Review by metalboyrick — APPROVED** [review]
LGTM!

**Pull request merged**